src/python/fuzzing/fuzzer_selection.py

```diff
@@ -80,10 +80,13 @@ def get_fuzz_task_payload(platform=None):
     queue_override = environment.get_value('QUEUE_OVERRIDE')
     platform = queue_override if queue_override else environment.platform()
 
-  query = data_types.FuzzerJob.query()
-  query = query.filter(data_types.FuzzerJob.platform == platform)
+  query = data_types.FuzzerJobs.query()
+  query = query.filter(data_types.FuzzerJobs.platform == platform)
+
+  mappings = []
+  for fuzzer_jobs in query:
```

Comment:
s/fuzzer_jobs/entity to avoid confusion with fuzzer_jobs.fuzzer_jobs.

Reply:
Done.